pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -54,7 +54,8 @@
       "StableVersions": [
         "4.0.1",
         "4.0.0",
-        "4.3.0"
+        "4.3.0",
+        "4.4.0"
```

Comment:
If you're going to do this, can you make sure you do it holistically with all the stable changes that were made in the release branch? That should be done in a separate commit.  At the same time you need to make sure you update the harvest project so that we download these for harvesting.